linters/terraform-docs/plugin.yaml

```diff
@@ -0,0 +1,21 @@
+version: 0.1
+lint:
+  definitions:
+    - name: terraform-docs
+      files: [terraform]
+      main_tool: terraform-docs
+      description: Generate terraform + tofu module documentation
+      commands:
+        - name: terraform-docs
+          output: rewrite
+          formatter: true
+          run_from: ${target_directory}
+          run: terraform-docs markdown --output-file=README.md .
```

Comment:
One note here: This is my team and I's (and I would suspect many many others) most common usage when using `terraform-docs` with terraform child + root modules. However, the tool is fairly configurable and [can be run in a lot of different ways](https://terraform-docs.io/user-guide/configuration/). I didn't see an obvious example of how make these `run` commands configurable, so I figured I'd put up this lightly working version for review + ask for feedback from the trunk team. 

Do ya'll have any other configurable plugins that I should check out for inspiration to make sure this doesn't need an immediate update from the 2nd person to use it?